web/client/epics/dashboard.js

```diff
@@ -187,7 +184,6 @@ module.exports = {
     // saving dashboard flow (both creation and update)
     saveDashboard: action$ => action$
         .ofType(SAVE_DASHBOARD)
-        .map(({resource}) => ({resource: cleanResource(resource)}))
```

Comment:
can you add a unit test for this epic, since i noticed it is missing?
i would check the presence of quickfilters and filter tabel for the table widget (at least)
thanks

Reply:
@MV88 
covered saveDashboard with tests. saveDashboard is not changing payload of action, but only making requests to the server, so the unit test to check quickFilters in the  payload is useless.